client/app/home/home.html

```diff
@@ -1,4 +1,19 @@
 <div class="container-fluid">
+    <oui-message class="d-block mt-4"
+                 data-ng-if="$ctrl.hasAScheduledMaintenanceOperation"
+                 data-type="info">
+        <p data-translate="homepage_scheduled_maintenance_operation_bhs3_info"></p>
+        <p>
+            <span data-translate="homepage_scheduled_maintenance_operation_bhs3_more_detail"></span>
+            <a href="http://travaux.ovh.net/?do=details&id=32795"
+               target="_blank"
+               rel="noopener">
+                <span class="sr-only" data-translate="common_new_window"></span>
+                http://travaux.ovh.net/?do=details&id=32795
```

Comment:
No variable to store it ? Or this could be in a `<span>` ?

Reply:
This is a temporary message that we want to show to the user but I agree with what you're saying